org-roam-db.el

```diff
@@ -227,10 +228,11 @@ This is equivalent to removing the node from the graph."
 
 (defun org-roam-db--insert-ref (file ref)
   "Insert REF for FILE into the Org-roam cache."
-  (org-roam-db-query
-   [:insert :into refs
-    :values $v1]
-   (list (vector ref file))))
+  (let ((key (elt (split-string ref ":") 1)))
```

Comment:
Won't this run into trouble with other types of links in the ROAM_KEY? e.g. `https://example.com` will get stripped to `//example.com`

Maybe we should check if the prefix is in `org-ref-cite-types` before splitting, and then strip off that prefix. Otherwise leave the key untouched.

Another issue this might run into is if a citekey has a ':' in it, this will strip off the stuff afterwards (after a quick search it appears that ':' is allowed in a bibtex citekey). e.g. in a pathological case: `cite:key:withcolon` would get  stripped to `key` instead of `key:withcolon`, or `key:withcolon` would get stripped to just `withcolon`.

Reply:
```suggestion
  (let ((key ref))
      (dolist (prefix org-ref-cite-types)
        (setq key (string-remove-prefix (concat prefix ":") key)))
```